src/components/FullPageWebView/FullPageWebView.js

```diff
@@ -0,0 +1,58 @@
+import { useFocusEffect, useNavigation, useRoute } from "@react-navigation/native";
+import { getAPIToken } from "components/LoginSignUp/AuthenticationService";
+import { ViewWrapper } from "components/SharedComponents";
+import React, { useState } from "react";
+import WebView from "react-native-webview";
+
+const FullPageWebView = () => {
```

Comment:
The designs for how the full page in-app browser should look like are here:
https://www.figma.com/file/MChpvx4ZrKEVVwsWKt4lkI/iNaturalist-Mobile-UI-Design?type=design&node-id=4292-26271&mode=design&t=P0wnKV1p3oYLhoTF-4